music_assistant/providers/internet-archive/manifest.json

```
@@ -0,0 +1,11 @@
+{
+  "domain": "internet-archive",
```

```suggestion
  "domain": "internet_archive",
```
use underscore please